draft-looker-oauth-attested-key-based-client-authentication.md

```diff
@@ -94,34 +94,51 @@ The following diagram depicts the conceptual interactions.
                      +------->|               |<------------------------>|               |
   (1) Generate       |        |    Client     |    (6) Interaction       | Authorization |
   Attestation Key    |        |   Instance    |    using Client          |    Server     |
-                     +--------|               |    Key Attestation + PoP |               |
+                     +--------|               |    Attestation + PoP     |               |
                               |               |    for authentication    |               |
                               +---------------+                          +---------------+
                                  / \      |
                                   |       |
                                   +-------+
                                 (5) Generate
-                              Key Attestation PoP
+                           Client Attestation PoP
 ~~~
 
-Note defining the protocol for steps 2 and 4, including how the client instance authenticates with the client backend is out of scope of this specification.
+The flow starts with the Authorization Server communicating its requirements for the client authentication to the client, made available in its metadata. A Client instance that wants to request an access token from this Authorization Server needs to obtain a Client Attestation JWT, for example, from its client backend first.
```

Comment:
This paragraph reads like there is a dependency on an AS providing a metadata document for a client to discover when in reality a client could know through an out of band mechanism that its what the AS requires for client authentication